### PR TITLE
Fix wrong type annotation

### PR DESCRIPTION
**Pull request by fabianhjr**
Fix wrong type annotation, floor should be float -> int

### CODE REVIEW

File: stdlib/2and3/math.pyi [hunk 1]
```diff
@@ -29,7 +29,7 @@ def exp(x: float) -> float: ...
 def expm1(x: float) -> float: ...
 def fabs(x: float) -> float: ...
 def factorial(x: int) -> int: ...
-def floor(x: float) -> float: ...
+def floor(x: float) -> int: ...
```
Comment (gvanrossum) [inline stdlib/2and3/math.pyi]: Actually it's float in PY2 and int in PY3...

Comment (fabianhjr) [inline stdlib/2and3/math.pyi]: :| weird, messed up then. Should I just copy this file to 3/ and change it there?

**Comment by gvanrossum** [comment]
You can use a check for sys.version_info. (Though beware that mypy doesn't support those yet (https://github.com/python/mypy/issues/698).

**Comment by fabianhjr** [comment]
Hey @gvanrossum, I did go around checking for that; the issue with sys.versio_info is that it won't pass the tests since python2.x and python3.x would need the same signature; So I went ahead and split the file in two.

**Comment by gvanrossum** [comment]
You can put `# type: ignore` on it to make the tests pass. Then we can take
that out when mypy implements version checking. Seems better than splitting
up the file.

**Comment by fabianhjr** [comment]
Ok, will do.

On Wed, Jul 20, 2016 at 11:45 PM, Guido van Rossum <notifications@github.com

**Comment by gvanrossum** [comment]
Suggestion for the purpose failure: put the ... on the next line?

--Guido (mobile)

**Comment by matthiaskramm** [comment]
We have a fix for that in pytype, as well. I'm the process of exporting it.

**Comment by gvanrossum** [comment]
@matthiaskramm, Can you re-run the failed test to confirm that it's fixed
and then merge?

**Comment by matthiaskramm** [comment]
Will do. Just waiting for the export to finish. (Exporting pytype takes a few hours, unfortunately)

**Comment by fabianhjr** [comment]
Thanks a lot @matthiaskramm C:

**Comment by matthiaskramm** [comment]
Closing and reopening, to (try to) force a Travis re-run.

**Pull request merged**